lib/matplotlib/tests/test_artist.py

```diff
@@ -152,30 +150,26 @@ def test_remove():
     im = ax.imshow(np.arange(36).reshape(6, 6))
     ln, = ax.plot(range(5))
 
-    assert_true(fig.stale)
-    assert_true(ax.stale)
+    assert fig.stale
```

Comment:
does py.test not have the nice convenience `assert_*` functions like `nose.tools`?